Assets/HoloToolkit-Examples/Input/Materials/HoloToolkit_Focusable.mat

```diff
@@ -8,7 +8,8 @@ Material:
   m_PrefabInternal: {fileID: 0}
   m_Name: HoloToolkit_Focusable
   m_Shader: {fileID: 4800000, guid: f1f3948b20c230044bda31b620ddd37f, type: 3}
-  m_ShaderKeywords: _EMISSION
+  m_ShaderKeywords: _EMISSION _SPECULARHIGHLIGHTS_ON _USEAMBIENT_ON _USEDIFFUSE_ON
```

Comment:
Does updating this material required for the gLTF update?

Reply:
The MotionControllerTest scene's cube is broken without it. I suppose it's not required for the glTF drop itself, but it fixes the main test scene for the glTF models.